nf_test/f03lib_f_interfaces.F90

```diff
@@ -307,6 +307,46 @@ End Function cmax_long
  cmax = ccmax
 
  End Function max_long
+!-------------------------------  min_longlong  ----------------------------------
```

Comment:
You should add your name to the list of authors at the top of the program, whenever you add this much code anywhere, or make extensive changes.